### ISSUE TITLE
"npm run nxt" on Ubuntu 18.04LTS wont run properly 

### ISSUE THREAD

**Issue by Casper0513** [sandoche/Proof-of-Stake-Cryptocurrency-generator#34]
Ubuntu 18.04LTS
Node: v13.2.0
Java: 11

Initializing Nxt server version 1.11.13
javafx not supported
isHeadless=false
Runtime mode nxt.env.CommandLineMode
User home folder /home/casper/Proof-of-Stake-Cryptocurrency-generator/nxt
Loading nxt-default.properties from classpath
Exception in thread "main" java.lang.ExceptionInInitializerError
Caused by: java.lang.RuntimeException: Using an nxt-default.properties file from a version other than 1.11.13 is not supported!!!
	at nxt.Nxt.<clinit>(Nxt.java:80)

**Comment by sandoche** [comment]
Is Java well installed ?

**Comment by Casper0513** [comment]
yes it is I actually got my own Blockchain created but the thing is How do I upload it to Github properly so that I can run it in DigitalOcean droplet Properly????? Help would be great thanks

**Comment by sandoche** [comment]
There are some info here:
https://github.com/sandoche/Proof-of-Stake-Cryptocurrency-generator#step-4---host-your-nodes-in-some-servers

Also you need to make sure to open the right ports.
